recipes/pyfftw/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.10.1" %}
+
+package:
+    name: pyfftw
+    version: {{ version }}
+
+source:
+    fn: pyFFTW-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/p/pyFFTW/pyFFTW-{{ version }}.tar.gz
+    md5: dab57de9d23f3b333115265914f3d240
+
+requirements:
+    build:
+        - cython
+        - python
+        - numpy x.x
+        - scipy      # For testing.
```

Comment:
> Yeah, it can't because they have to be run at build time. Tests don't get installed.

I thought it was something like that.

> Though I am having trouble getting the Python tests to run due to some import errors. It seems to think it can't import things from the test directory. Not sure why as it works outside of `conda`. If you have any ideas, I would certainly appreciate the help.

I will take a look tomorrow. Not sure what is going on.